src/Development/IDE/GHC/Orphans.hs

```diff
@@ -1,8 +1,10 @@
 -- Copyright (c) 2019 The DAML Authors. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
+{-# LANGUAGE CPP               #-}
```

Comment:
Hlint is failing because we explicitly whitelist modules that are allowed to use CPP. You can just add this module to the list: https://github.com/digital-asset/ghcide/blob/9adb11125ed8c671e802405a272c5c34ec1df0d3/.hlint.yaml#L77

Reply:
Thanks! Should be fixed now.